db/migrate/20200802222331_create_payments.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePayments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :payments, id: :uuid do |t|
+      t.date :paid_on
+      t.date :care_started_on
+      t.date :care_finished_on
+      t.monetize :amount
```

Comment:
I don't know that we'll ever want to accept a payment of less than 1 cent, so we should put that validation in here for the amount being greater than 0?  I'm not sure if monetize can do that, but if we could do it at the db level and on the model, that'd be amazing.